### PR TITLE
Show opened bugs on Component Readiness test details page

### PR DESCRIPTION
**Pull request by stbenjam**
[TRT-1238](https://issues.redhat.com//browse/TRT-1238)

This [test](https://sippy.dptools.openshift.org/sippy-ng/component_readiness/test_details?arch=amd64&baseEndTime=2023-05-16%2023%3A59%3A59&baseRelease=4.13&baseStartTime=2023-04-18%2000%3A00%3A00&capability=Other&component=oc&confidence=95&environment=ovn%20upgrade-micro%20amd64%20gcp%20standard&excludeArches=arm64%2Cheterogeneous%2Cppc64le%2Cs390x&excludeClouds=openstack%2Calibaba%2Cibmcloud%2Clibvirt%2Covirt%2Cunknown&excludeVariants=hypershift%2Cosd%2Cmicroshift%2Ctechpreview%2Csingle-node%2Cassisted%2Ccompact&groupBy=cloud%2Carch%2Cnetwork&ignoreDisruption=true&ignoreMissing=false&minFail=3&network=ovn&pity=5&platform=gcp&sampleEndTime=2023-09-29%2023%3A59%3A59&sampleRelease=4.14&sampleStartTime=2023-09-22%2000%3A00%3A00&testId=openshift-tests%3A1894de0e75f76589f6c8ae8bae68d6d7&testName=[sig-cli]%20oc%20adm%20role-selectors%20[apigroup%3Atemplate.openshift.io]%20[Suite%3Aopenshift%2Fconformance%2Fparallel]&upgrade=upgrade-micro&variant=standard) report now looks like:

image

### CODE REVIEW

File: sippy-ng/src/component_readiness/CompReadyTestReport.js [hunk 1]
```diff
@@ -304,8 +305,13 @@ export default function CompReadyTestReport(props) {
           &gt; {testName}{' '}
         </Link>
       </h3>
-      Test Name: {testName}
-      <hr />
+      <div align="center" style={{ marginTop: 50 }}>
+        <h2>{testName}</h2>
+      </div>
+      <h2>Known Bugs</h2>
```
Comment (xueqzhan) [inline sippy-ng/src/component_readiness/CompReadyTestReport.js:311]: Does it make it clearer if this is center aligned?

**Review by xueqzhan — COMMENTED** [review]
Really nice to have this.

File: sippy-ng/src/component_readiness/CompReadyTestReport.js (hunk 1 above)
Comment (stbenjam) [reply sippy-ng/src/component_readiness/CompReadyTestReport.js:311]: I don't think so, section headers are left-aligned in Sippy (i.e. [Test Analysis](https://sippy.dptools.openshift.org/sippy-ng/tests/4.15/analysis?test=OSD%20e2e%20suite.%5BSuite%3A%20operators%5D%20%5BOSD%5D%20Must%20Gather%20Operator%20clusterRoles%20should%20exist&filters=%7B%22items%22%3A%5B%7B%22columnField%22%3A%22name%22%2C%22operatorValue%22%3A%22equals%22%2C%22value%22%3A%22OSD%20e2e%20suite.%5BSuite%3A%20operators%5D%20%5BOSD%5D%20Must%20Gather%20Operator%20clusterRoles%20should%20exist%22%7D%2C%7B%22columnField%22%3A%22variants%22%2C%22not%22%3Atrue%2C%22operatorValue%22%3A%22contains%22%2C%22value%22%3A%22never-stable%22%7D%2C%7B%22columnField%22%3A%22variants%22%2C%22not%22%3Atrue%2C%22operatorValue%22%3A%22contains%22%2C%22value%22%3A%22aggregated%22%7D%5D%2C%22linkOperator%22%3A%22and%22%7D)).  CR isn't exactly homogenous with the rest but I'd prefer to stay consistent.

**Pull request merged**